src/Components/profileOutline.js

```diff
@@ -0,0 +1,208 @@
+// code here! 
+// beep boop beep boop
+import PropTypes from 'prop-types';
+
+import { useEffect } from "react";
+
+const styles = {
+    header: {
+        fontSize: 69,
+        color: "maroon",
+        fontWeight: 900,
+    }, 
+    text: {
+        fontSize: 69,
+        color: "brown",
+    },
+};
+
+
+export default function ProfileOutline({
+    name, 
+    title,
+    pronouns,
+    email,
+    phone,
+    city,
+    state,
+    ethnicity,
+    age,
+    genderIdentity,
+    sexuality,
+    veteran,
+    disability,
+    housingSituation,
+    employmentStatus,
+    priorConvictions,
+}
+){
+// some code here 
+// const user = {
+//     name: 'kaylee',
+//     title: 'administrator',
+//     pronouns: 'she/her',
+//     email: 'kaeleytran@gmail.com', 
+// };
+
+useEffect(()=> {
+// do something 
+},//set something 
+);
+
+return (
+<div>
+    <div style={styles.header}>
+        Name:
+    </div>
```

Comment:
We should also talk to the designers, but I think we can make a separate component for demographic info, which will then be used by the profile component!